src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
I had a change of heart about this over the weekend. I think you are right - let's leave it as is. We don't need to convert it to programmatic config. I was too eager to convert everything.